live/README.md

```diff
@@ -63,3 +63,43 @@
    machine start livecd
    machine gui livecd
     ```
+
+* Build a provisioning ISO
+
+  ```
+  $ stacker build -f provision-stacker.yaml --layer-type=squashfs
+  $ ./build-livecd-rfs --layer oci:oci:provision-rootfs-squashfs \
+     --output provision.iso
+  $ mkdir SUDI; cp ~/.local/share/machine/trust/keys/snakeoil/manifest/default/sudi/XXX/* SUDI/
```

Comment:
XXX?

Reply:
Well, the machine uuid, e.g. 4cc76b82-948c-44b8-948c-1cc9a7d460d0 .

If you do a 'keyctl project add snakeoil SN01', then it'll be SN01.

In the end this needs to be also tied to the board_serial DMI info that we pass to the machine initialization.